test/plugin/test_buffer_file_chunk.rb

```diff
@@ -768,4 +771,83 @@ def gen_chunk_path(prefix, unique_id)
       end
     end
   end
+
+  sub_test_case 'compressed buffer' do
+    setup do
+      @src = 'text data'
+      @gzipped_src = compress(@src)
+    end
+
+    test '#bytesize returns a size of the compressed string' do
+      src = 'テキストデータ text data'
```

Comment:
Please avoid to use Japanese characters. It makes hard to edit these test cases for people not using Japanese.